SpeechResponder/Cottle.xshd

```diff
@@ -10,7 +10,7 @@
   <Color name="Body text" foreground="DarkSlateGray" exampleText="This is body text"/>
   <Color name="Comment" foreground="Green" fontStyle="italic" exampleText="{_ This is a comment. }"/>
   <Color name="Code" foreground="Purple" exampleText="{ variableName }"/>
-  <Color name="Delimiter" foreground="Purple" fontWeight="bold" exampleText="{ | }"/>
+  <Color name="Delimiter" foreground="Purple"  background="LightGreen" fontWeight="bold" exampleText="{ | }"/>
```

Comment:
I'm not sure about this one.

Reply:
Whoops, that was from testing last night. Forgot to revert that change.

Reply:
Fix pushed.